customTests/azure_gpu_bandwidth.nhc

```diff
@@ -316,7 +316,7 @@ function check_rocm_gpu_bw(){
     done    
 
     for bad_result in "${bad_results[@]}"; do
-        die 1 "check_gpu_bw: $bad_result"
+        die 1 "check_gpu_bw: $bad_result. FaultCode: NHCNA"
```

Comment:
This is an actual failure for GPU Bandwidth

Reply:
Will reassign a new fault code